e2e/cypress.json

```diff
@@ -31,8 +33,9 @@
         "resetBeforeTest": false,
         "runLDAPSync": true,
         "secondServerURL": "http://localhost/s/p",
+        "serverEdition": "Team",
         "smtpUrl": "http://localhost:10080",
         "storybookUrl": "http://localhost:6006/",
         "webhookBaseUrl": "http://localhost:3000"
     }
-}
+}
```

Comment:
Please revert this since they were the same as 'master' branch. You may update this branch to take effect the latest.

Reply:
```suggestion
}

```